docs/QUICKSTART.md

```diff
@@ -0,0 +1,145 @@
+# Quickstart
+
+You will need `docker`, `cargo`, `make`, `kind`, and the `aws` and `eksctl` CLIs for this.
+Caution: if you follow these instructions, you will create an EKS cluster and EC2 instances!
+
+Set the `TESTSYS_DIR` variable to point to the directory in which you have cloned the project.
+For example:
+
+```shell
+export TESTSYS_DIR="${HOME}/repos/bottlerocket-test-system"
+```
+
+Set a few more variables.
+You might want to change the AWS region:
+
+```shell
+export TESTSYS_CLUSTER_NAME=testsys
+export EKS_CLUSTER_NAME=external-cluster
+export KUBECONFIG="/tmp/${TESTSYS_CLUSTER_NAME}.yaml"
+export EKS_REGION="us-west-2"
+export CARGO_HOME="${TESTSYS_DIR}/.cargo"
+K8S_VER="1.21"
+REGION="${EKS_RESION}"
+alias testsys="${TESTSYS_DIR}/.cargo/bin/testsys"
+```
+
+Install the `testsys` command line tool into the local CARGO_HOME and build the containers we need:
+
+```shell
+cd "${TESTSYS_DIR}"
+cargo install --path "${TESTSYS_DIR}/bottlerocket/testsys" --force
+
+make controller
+make ec2-resource-agent
+make eks-resource-agent
+make sonobuoy-test-agent
+
+docker tag controller controller:eks
+docker tag ec2-resource-agent ec2-resource-agent:eks
+docker tag eks-resource-agent eks-resource-agent:eks
+docker tag sonobuoy-test-agent sonobuoy-test-agent:eks
+```
+
+We will use a local `kind` cluster as our TestSys cluster.
+Here we create it and load our container images into it using `kind`.
+Note, the `kind load docker-image` command frequently reports an error even when everything seems to have worked.
+
+```shell
+kind create cluster --name "${TESTSYS_CLUSTER_NAME}"
+
+kind load docker-image \
+  controller:eks \
+  ec2-resource-agent:eks \
+  eks-resource-agent:eks \
+  sonobuoy-test-agent:eks \
+  --name "${TESTSYS_CLUSTER_NAME}"
+```
+
+Next we install the TestSys namespace, controller and CRD schemas into the TestSys cluster.
+We also set our kubeconfig context to the testsys namespace for convenience.
+
+```shell
+testsys install --controller-uri controller:eks
+kubectl config set-context --current --namespace="testsys-bottlerocket-aws"
+```
+
+We will be creating an EKS cluster and EC2 instances, so we need to create a Kubernetes secret with our AWS credentials.
+
+```shell
+testsys add secret map  \
+ --name "aws-creds" \
+ "access-key-id=$(aws configure get default.aws_access_key_id)" \
+ "secret-access-key=$(aws configure get default.aws_secret_access_key)"
+```
+
+Now we are ready to run a Bottlerocket test.
+We get the latest AMI ID.
+Then we pass it to TestSys which will create an EKS cluster, launch Bottlerocket nodes and run a Sonobuoy test in 'quick' mode.
+
+**Caution**: The command below specifies `never` as the `--cluster-destruction-policy`.
+This is because creating a cluster takes a long time, and we might want to re-use it.
+To delete the cluster manually, use `eksctl delete cluster "external-cluster"` or delete the relevant CloudFormation stacks.
```

Comment:
We have instructions to delete clusters with `eksctl` here but no where before this do we provide instructions on creating the cluster via `eksctl`, we only have instructions to create the cluster locally with `kind`.

Reply:
I believe we are able to create the cluster with `testsys`, but as far as I know we don't have anything built in to perform the clean up. So we do need to perform the clean up manually with `eksctl` or go through and delete things from the console.

Reply:
Also to point out - this is just moved out of the base README file, so this is what we currently have published.